demos/ASP.Net Core/ITOps.ViewModelComposition.Gateway/ComposableRouteHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Gateway
+{
+    class ComposableRouteHandler
+    {
+        public static async Task HandleGetRequest(HttpContext context)
+        {
+            var result = await CompositionHandler.HandleGetRequest(context);
+            if (result.StatusCode == StatusCodes.Status200OK)
+            {
+                string json = JsonConvert.SerializeObject(result.ViewModel, GetSettings(context));
+                context.Response.ContentType = "application/json; charset=utf-8";
+                await context.Response.WriteAsync(json);
+            }
+            else
+            {
+                context.Response.StatusCode = result.StatusCode;
+            }
+        }
+
+        static JsonSerializerSettings GetSettings(HttpContext context)
+        {
+            if (!context.Request.Headers.TryGetValue("Accept-Casing", out StringValues casing))
```

Comment:
Thinking more about it, in theory we could entirely remove it. I'll spike it in another branch.